docs/eligible-collateral-representation.md

```diff
@@ -220,13 +220,13 @@ type AssetCriteria:
   agencyRating AgencyRatingCriteria (0..*)
   maturityType MaturityTypeEnum (0..1)
   maturityRange PeriodRange (0..1)
-  assetIdentifier AssetIdentifier (0..*)
+  specificAssets Asset (0..*)
```

Comment:
The design principles impose a singular naming convention even when we are expecting lists of data i.e. `specificAsset` should be used here. I think adding the 's' on the end makes more sense in this instance.